js-miniapp-sdk/CHANGELOG.md

```diff
@@ -15,7 +15,7 @@
 
 ### 1.3.0 (2020-10-22)
 
-- **Feature:** Added support for requesting the load and display of Interstitial & Rewarded Ads in Host app. [See here](README.md#4-Show-Ads).
+- **Feature:** Added support for requesting the load and display of Interstitial, Rewarded & Banner Ads in Host app. [See here](README.md#Show-Ads).
```

Comment:
You should add Banner ads as a new entry under `1.7.0`.

Reply:
Put that under 1.7.0

Reply:
Sorry genuinely my bad